### PR TITLE
Automate CLI doc generation for gitbook

### LINKED ISSUE #1: Devs SBAT update CLI docs automatically
### Expected Behavior

1. Check if CLI package has changed using git pre-commit hook
2. Generate README and command docs with `oclif-dev readme --multi`
3. Check in updates to gitbook `docs/` directory

### Current Behavior

Manual gitbook `docs/` updates

### LINKED ISSUE #19: Devs SBAT update CLI docs automatically
### Expected Behavior

1. Check if CLI package has changed using git pre-commit hook
2. Generate README and command docs with `oclif-dev readme --multi`
3. Check in updates to gitbook `docs/` directory

### Current Behavior

Manual gitbook `docs/` updates

### PR DESCRIPTION
**Pull request by yorhodes**
### Description

Allows `packages/docs/command-line-interface` to be populated by `yarn docs` in `packages/cli`.

Adds CI for checking gitbook markdown syntax in `build-all-packages` job. 
Adds CI for checking if `docs/command-line-interface` should be updated/have been checked in.

### Tested

Checked in markdown is result of `yarn docs`

Used 
`circleci local execute --job build-all-packages -v ~/celo/celo-monorepo:/home/circleci/app
` to check that changes to `build-all-packages` work as expected.

### Related issues

- Fixes #1

### CODE REVIEW

**Review by nambrot — DISMISSED** [review]

**Review by ashishb — DISMISSED** [review]
Next time consider splitting unrelated changes into multiple PRs

**Review by ashishb — DISMISSED** [review]

File: packages/docs/command-line-interface/node.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+### Accounts
+
+List node accounts
+
+```
+USAGE
+  $ celocli node:accounts
+```
+
+_See code: [src/commands/node/accounts.ts](https://github.com/celo-org/celo-monorepo/tree/master/packages/cli/src/commands/node/accounts.ts)_
```
Comment (timmoreton) [inline packages/docs/command-line-interface/node.md]: Nit: would be nice to display the `packages/cli` prefix on these paths

**Review by timmoreton — DISMISSED** [review]
LGTM!

**Comment by timmoreton** [comment]
can you add instructions in the top-level README about how to build the cli docs before check-in?

**Review by timmoreton — APPROVED** [review]

**Pull request merged**